units/container-getty@.service.in

```diff
@@ -25,10 +25,9 @@ Conflicts=rescue.service
 Before=rescue.service
 
 [Service]
-# The '-o' option value tells agetty to replace 'login' arguments with an
-# option to preserve environment (-p), followed by '--' for safety, and then
-# the entered username.
-ExecStart=-/sbin/agetty -o '-p -- \\u' --noreset --noclear - ${TERM}
+# The '-o' option value tells agetty to replace 'login' arguments with '--' for
+# safety, and then # the entered username.
```

Comment:
`#` should be dropped??

Reply:
Uh, still there...

Reply:
Guh, sorry, something is wonky with my editor's comment toggle function.

Should all be fixed now for real.